net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils autotools
+
+DESCRIPTION="Facebook protocol plugin for BitlBee"
+HOMEPAGE="https://github.com/bitlbee/bitlbee-facebook"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug"
+
+if [ "${PV}" = "9999" ] ; then
```

Comment:
Same as the last. Eliminate the conditional.

Reply:
Leave it, standard Gentoo idiom